### PR TITLE
[main] <-- [develop] First version of the RandomPasswordApp

### PR DESCRIPTION
**Pull request by Pawlitka**

### CODE REVIEW

**Comment by Pawlitka** [comment]
IF THE REVIEW WILL BE COMPLETED. CLOSE THE PULL REQUEST AND CREATE A NEW ONE TO MERGE TO RELEASE, NOT MAIN.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Random Password Generator</title>
```
Comment (Cozoob) [inline index.html]: Should be label :)

You should add this text in js. You can make it using id.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="container">
+        <h1>Generate a<br><span> RANDOM PASSWORD</span></h1>
```
Comment (Cozoob) [inline index.html]: Should be labels

File: script.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    password += number[Math.floor(Math.random() * number.length)];
+    password += symbol[Math.floor(Math.random() * symbol.length)];
+
+    while(length > password.length) {
+        password += allChars[Math.floor(Math.random() * allChars.length)];
+    }
+    passwordBox.value = password;
+}
+
+function copyPassword(){
```
Comment (Cozoob) [inline script.js]: It is event handler... so rename it to `handleOnClickCopyPassword`

File: index.html [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+        <div class="display">
+          <input type="text" id="password" placeholder="password">
+            <img src="copy.png" alt="copy.png" onclick="copyPassword()">
+        </div>
+        <button onclick="createPassword()"><img src="generate.png" alt="generate.png">generate password</button>
```
Comment (Cozoob) [inline index.html]: Should be label

File: script.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
+const passwordBox = document.getElementById("password");
+const length = 15;
+const upperCase = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+const lowerCase = "abcdefhijklmnopqrstuvwxyz";
+const number = "0123456789";
+const symbol = "!@#$%^&*()_+~";
+const allChars = upperCase + lowerCase + number + symbol;
```
Comment (Cozoob) [inline script.js]: Const names should be UPPER CASE and snake_case so... `PASSWORD_BOX` etc.

**Review by Cozoob — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Pawlitka) [reply index.html]: done. :)

File: index.html (hunk 2 above)
Comment (Pawlitka) [reply index.html]: Done !!

File: script.js (hunk 3 above)
Comment (Pawlitka) [reply script.js]: Okay done.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title id="title">Random Password Generator</title>
+        <link rel="stylesheet" href="styles.css">
+    </head>
+    <body>
+        <div class="container" onclick="test()">
```
Comment (Cozoob) [inline index.html]: remove onclick="test()"

File: script.js [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+};
+const LABELS = {
+    [ELEMENTS_IDS.title]: "Random Password Generator",
+    [ELEMENTS_IDS.headerPrefix]: "Generate a",
+    [ELEMENTS_IDS.headerSuffix]: "RANDOM PASSWORD",
+    placeholderPassword: "password",
+    [ELEMENTS_IDS.buttonGeneratePassword]: "Generate password"
+};
+
+function createPassword(){
```
Comment (Cozoob) [inline script.js:25]: create extra function called `handleClickButtonGeneratePassword`
and inside of the handler, call `createPassword();`

```javascript
function handleClickButtonGeneratePassword(){
  createPassword();
}
```

File: script.js [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+    password += NUMBER[Math.floor(Math.random() * NUMBER.length)];
+    password += SYMBOL[Math.floor(Math.random() * SYMBOL.length)];
+
+    while(LENGTH > password.length) {
+        password += ALL_CHARS[Math.floor(Math.random() * ALL_CHARS.length)];
+    }
+    PASSWORD_BOX.value = password;
+}
+
+function handleOnClickCopyPassword(){
```
Comment (Cozoob) [inline script.js]: rename to `handleClickCopyPassword`

**Review by Cozoob — CHANGES_REQUESTED** [review]

File: script.js (hunk 8 above)
Comment (Pawlitka) [reply script.js]: Done

File: index.html (hunk 6 above)
Comment (Pawlitka) [reply index.html]: done

File: script.js (hunk 7 above)
Comment (Pawlitka) [reply script.js:25]: doneee

File: script.js [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+    headerSuffix: "headerSuffix",
+    buttonGeneratePassword: "buttonGeneratePassword"
+};
+const LABELS = {
+    [ELEMENTS_IDS.title]: "Random Password Generator",
+    [ELEMENTS_IDS.headerPrefix]: "Generate a",
+    [ELEMENTS_IDS.headerSuffix]: "RANDOM PASSWORD",
+    placeholderPassword: "password",
+    [ELEMENTS_IDS.buttonGeneratePassword]: "Generate password"
+};
```
Comment (Cozoob) [inline script.js:20]: add extra space

Comment (Cozoob) [reply script.js:20]: and well you can merge :)

**Review by Cozoob — APPROVED** [review]

**Pull request closed without merging**